okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,10 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
```

Comment:
Lots of different words for the same thing: StackTrace, AllocationSite, ExecutionSite. I like ExecutionSite or StackTrace, but we should be consistent.